{
  "comments": [
    {
      "robotId": "Pylint/warning/attribute-defined-outside-init",
      "robotRunId": "5710542714961920",
      "properties": {
        "tricium_comment_uuid": "c10f08ee-0141-47f5-bd91-928474e14eb6"
      },
      "fixSuggestions": [],
      "key": {
        "uuid": "bd68e7c2_528a5077",
        "filename": "appengine/findit/model/flake/analysis/master_flake_analysis.py",
        "patchSetId": 7
      },
      "lineNbr": 384,
      "author": {
        "id": 1242318
      },
      "writtenOn": "2018-09-14T01:07:26Z",
      "side": 1,
      "message": "Attribute \u0027build_id\u0027 defined outside __init__.\nTo disable, add: # pylint: disable\u003dattribute-defined-outside-init",
      "tag": "autogenerated:tricium",
      "revId": "573ef4a9a45606c545d8cea064ebf056f8b07c36",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": false
    }
  ]
}